pkg/manager/member/tikv_member_manager.go

```diff
@@ -632,6 +633,11 @@ func (m *tikvMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 	if err != nil {
 		return err
 	}
+
+	if !upgrading && tc.Status.TiKV.Phase == v1alpha1.UpgradePhase {
+		endEvictLeader(m.deps, tc, helper.GetMinPodOrdinal(*set.Spec.Replicas, set))
```

Comment:
We should check the error returned here.

Reply:
fixed